contracts/liquidator/Liquidator.sol

```diff
@@ -12,7 +12,12 @@ import "./vendor/@uniswap/v3-periphery/contracts/interfaces/ISwapRouter.sol";
 import "../CometInterface.sol";
 import "../ERC20.sol";
 
+/**
+ * @title Compound's Liquidator Contract
+ * @author Compound
+ */
```

Comment:
Maybe "Compound's Example Liquidator Contract" and possibly a note that this is meant for others to build upon?

Reply:
Sounds good, I will update it